app/api/v1/endpoints/memory/endpoints.py

```
@@ -425,6 +491,151 @@ async def list_relationships(
 
+@router.get(
+    "/graph",
+    response_model=GraphDataResponse,
+    summary="Get graph data",
+    description=(
+        "Returns graph-ready nodes/links for Memory UI including edited-memory "
+        "influence metadata for trust/explainability."
+    ),
+)
+async def get_graph_data(
+    current_user: Annotated[TokenPayload, Depends(get_current_user)],
```

Ruff (0.14.14)

[warning] Unused function argument: `current_user`

(ARG001)